### PR TITLE
Cleanup IconHandle nodes

### PR DESCRIPTION
**Pull request by briansull**
Removed unused fields of Icon nodes:
 - The union of gtIconHdl1, gtIconHdl2 and gtIconCPX, gtIIconCls is not used

Removed unused handle args from gtNewIconHandleNode and gtNewIcomEmbHndNode

### CODE REVIEW

**Comment by briansull** [comment]
@dotnet-bot Test tizen armed Cross Debug Build

**Comment by briansull** [comment]
@dotnet-bot Test tizen armel Cross Debug Build

**Comment by briansull** [comment]
@dotnet/jit-contrib PTAL

File: src/jit/compiler.hpp [hunk 1]
```diff
@@ -1052,9 +1051,6 @@ inline GenTreePtr Compiler::gtNewIconHandleNode(
 
 #if defined(LATE_DISASM)
     node = new (this, LargeOpOpcode()) GenTreeIntCon(TYP_I_IMPL, value, fields DEBUGARG(/*largeNode*/ true));
-
-    node->gtIntCon.gtIconHdl.gtIconHdl1 = handle1;
```
Comment (BruceForstall) [inline src/jit/compiler.hpp]: Do we still need this ifdef at all? Can we create a non-large node in all cases?

Comment (briansull) [reply src/jit/compiler.hpp]: I plan on investigating this further and fixing it in an upcoming change

**Pull request merged**